.github/workflows/push.yaml

```diff
@@ -194,6 +194,58 @@ jobs:
           name: macos-sparkle-update
           path: ${{ github.workspace }}/output
 
+  create-windows-update:
+    name: Create Windows Update 🎙️
+    if: github.repository_owner == 'obsproject' && github.ref_type == 'tag'
+    runs-on: windows-2022
+    needs: build-project
+    strategy:
+      fail-fast: false
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: "repo"
+          fetch-depth: 0
+          ref: ${{ github.ref }}
+
+      - name: Set Up Environment 🔧
+        id: setup
+        shell: bash
+        run: |
+          channel='stable'
+          if [[ ${GITHUB_REF_NAME} =~ (beta|rc) ]]; then
```

Comment:
Aren't we always escaping the first token to the test operation in Bash? IIRC it's not required in ZSH but in Bash you should still always double-quote the first token to avoid automatic tokenisation (if the variable contains spaces the effective code would be `[[ some value =~ (beta|rc) ]]` and that's invalid shell code.

Reply:
I think that is correct, though git refs shouldn't be able to have spaces. I haven't bothered rewriting this to powershell yet, so it was mostly just a stripped down version of what's used for the appcast.